test/tests/rails52.rb

```diff
@@ -13,7 +13,7 @@ def expected
       :controller => 0,
       :model => 0,
       :template => 5,
-      :generic => 16
+      :generic => 17
```

Comment:
Why is it changed to 17?

Reply:
These numbers indicate how many issues are in the test suite. Since this change added a test case, the expected number of issues increased.